coverage_assessment.R

```diff
@@ -41,10 +41,39 @@ nut <- get_data()
 
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
+library(dplyr)
+
+## Load functions in R directory ----
+for (f in list.files("R", full.names = TRUE)) source (f)
+
+
+# Read data ----
+nut <- get_data()
+
+
+with(nut, sum(muac >= 11.5 & muac <12.5 & oedema == 2, na.rm= TRUE))
+
+##number of moderate acute malnourished (MAM) cases children 6-59 months old 
+
+
+with(nut, sum(muac >= 11.5 & muac <12.5 & oedema == 2 & sex == 1 , na.rm= TRUE ))
+
+##number of moderate acute malnourished (MAM) male 6-59 months old 
+
+
+with(nut, sum(muac >= 11.5 & muac <12.5 & oedema == 2 & sex == 2 , na.rm= TRUE ))
+
+##number of moderate acute malnourished (MAM) female 6-59 months old 
+
+
+with(nut, sum(muac >= 11.5 & muac <12.5 & oedema == 2 & county == "Urban Montserrado" , na.rm= TRUE))
 
+##number of moderate acute malnourished (MAM) cases children 6-59 months old  in Urban Montserrado
 
 
+with(nut, sum(muac >= 11.5 & muac <12.5 & oedema == 2 & county != "Urban Montserrado" , na.rm= TRUE))
 
+##number of moderate acute malnourished (MAM) cases children 6-59 months old other than Urban Montserrado
```

Comment:
Both these are correct. Well done!

You may want to save this value into an object so that you can use it later on when you need to calculate prevalence?

Reply:
I forgot to add to my comment that you will need to take into account the age criteria as well. The children needs to be 6 months up to 59 months old.